torch/_dynamo/variables/builder.py

```diff
@@ -471,6 +472,11 @@ def build_key_value(i, k, v):
         elif isinstance(value, enum.Enum):
             self.install_guards(GuardBuilder.ID_MATCH)
             return EnumVariable(value=value, source=self.source)
+        elif DebuggingVariable.is_reorderable_logging_function(value):
+            # Put this above builtin_callable so that print() can be handled
+            # along with other builtin debugging functions
+            self.install_guards(GuardBuilder.BUILTIN_MATCH)
```

Comment:
You've made the list of reorderable logging functions customizable, why is BUILTIN_MATCH the correct guard to use here then?

Reply:
This can be ```FUNCTION_MATCH``` as well, but they are equivalent IIFC.